<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FlowNetworkAlgorithmWindow</class>
 <widget class="QWidget" name="FlowNetworkAlgorithmWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>931</width>
    <height>616</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FlowNetworkAlgorithmWindow</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>871</width>
     <height>571</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="1">
     <widget class="QCommandLinkButton" name="nextStepButton">
      <property name="text">
       <string>Następny krok</string>
      </property>
      <property name="description">
       <string>Wykonuje kolejny etap algorytmu</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="GraphView" name="mainNetworkView">
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="frameShape">
       <enum>QFrame::WinPanel</enum>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QCommandLinkButton" name="finishAlgorithmButton">
      <property name="text">
       <string>Skończ algorytm</string>
      </property>
      <property name="description">
       <string>Wykonuje algorytm do końca</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="GraphView" name="residualNetworkView">
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="frameShape">
       <enum>QFrame::WinPanel</enum>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Sieć  residaulna</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Sieć przepływowa</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QCommandLinkButton" name="saveResultButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Zapisz wynik</string>
      </property>
      <property name="description">
       <string>Zapisuje otrzymany graf do pliku</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2" rowspan="3">
     <widget class="QTextBrowser" name="progressConsole">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>431</width>
        <height>999</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="lineWidth">
       <number>2</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GraphView</class>
   <extends>QGraphicsView</extends>
   <header location="global">GraphView.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
